packages/mobile/src/app/selectors.ts

```diff
@@ -107,3 +107,5 @@ export const multiTokenUseUpdatedFeedSelector = (state: RootState) =>
 export const linkBankAccountEnabledSelector = (state: RootState) => state.app.linkBankAccountEnabled
 
 export const sentryTracesSampleRateSelector = (state: RootState) => state.app.sentryTracesSampleRate
+
+export const dappsListApiUrlSelector = (state: RootState) => state.app.dappListApiUri
```

Comment:
nit: So should we use `uri` or `url`? 😄 
Looks like we've already mixed both, seems `url` should be preferred in my opinion.